src/_guides/libraries/useful-libraries.md

```diff
@@ -138,20 +140,20 @@ Use the following resources to learn more about libraries and library packages.
 
 * [A Tour of the Dart Libraries](/guides/libraries/library-tour), which
   gives examples of many commonly used dart:* APIs.
-* [Unboxing Packages](https://news.dartlang.org/search/label/Unboxing%20Packages)
-  posts, written by written by Natalie Weizenbaum and published on
-  [news.dartlang.org.](https://news.dartlang.org/)
+* [Unboxing Packages]({{site.news}}/search/label/Unboxing%20Packages)
+  posts, written by written by Natalie Weizenbaum and published on the
+  [Dart news site.]({{site.news}})
   This page links to some of Natalie's posts, but she covers other packages
   not mentioned here, such as stream_channel, vm_service_client, and json_rpc_2.
 
 ### Packages contributed by the community
 
-* [pub.dartlang.org]({{site.pub}})
+* [Package site]({{site.pub}})
 
 ### API reference documentation
 
-* [api.dartlang.org]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}) contains the generated docs for dart:* libraries.
-* [pub.dartlang.org]({{site.pub}}) hosts the generated docs for published packages.
-* [docs.flutter.io](https://docs.flutter.io/) contains the generated docs for Flutter
+* [Dart API]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}) contains the generated docs for dart:* libraries.
```

Comment:
We need to rephrase these, since we're no longer spelling out the URLs. Alternatively, use the macro on the left side, as well as the right.